resources/css/app/_action-column.css

```
@@ -0,0 +1,89 @@
+    }
+
+    th.picture-column {
+        width: 1px;
+    }
+
+    th.action-column {
+        width: 1px;
+    }
+}
+
+@media (width >= theme(--breakpoint-lg)) {
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)